core/src/main/java/hudson/model/User.java

```diff
@@ -655,7 +655,7 @@ private boolean relatedTo(@Nonnull AbstractBuild<?,?> b) {
     @SuppressWarnings("unchecked")
     @WithBridgeMethods(List.class)
     public @Nonnull RunList getBuilds() {
-        return RunList.fromJobs(Jenkins.getInstance().allItems(Job.class)).filter(new Predicate<Run<?,?>>() {
+        return RunList.fromJobs((Iterable)Jenkins.getInstance().allItems(Job.class)).filter(new Predicate<Run<?,?>>() {
```

Comment:
I had tried quite a few combinations before finding that one, for instance with `Run` only:

![image](https://cloud.githubusercontent.com/assets/223853/24515358/a9dcacf2-1577-11e7-95fb-6eae5f7d1aa1.png)

So I'm ready to change to something else, but I thought the fix I have was readable/simple enough anyway (?).